hollow-ui-tools/src/main/java/com/netflix/hollow/ui/HollowUISession.java

```diff
@@ -85,24 +87,23 @@ public static HollowUISession getSession(HttpServletRequest req, HttpServletResp
     }
 
     static {
-        Thread sessionCleanupThread = new Thread(new Runnable() {
-            public void run() {
-                Iterator<Map.Entry<Long, HollowUISession>> iter = sessions.entrySet().iterator();
-                while(iter.hasNext()) {
-                    Map.Entry<Long, HollowUISession> entry = iter.next();
-                    if(entry.getValue().lastAccessed + SESSION_ABANDONMENT_MILLIS < System.currentTimeMillis())
-                        iter.remove();
-                }
-
-                try {
-                    Thread.sleep(60000L);
-                } catch (InterruptedException e) { }
-            }
-        });
-
+        Thread sessionCleanupThread = new Thread(HollowUISession::cleanupSessions, "hollow | ui-session-cleanup");
```

Comment:
Minor: might be nice to refactor thread naming into a static method to make it easier to make them consistent. Also I'm not so sure about using vertical bar as a separator - maybe there are log systems using it as a separator, how about just `hollow-`?

Reply:
Let's create a method that accepts a String and returns a new daemon thread. That method could then be reused in many places. That method could accept a class associated with the thread as well as a name from which the thread name is composed. e.g.:

```java
Thread createDaemonThread(Runnable r, Class<?> context, String description) {
    Thread t = new Thread(r, createThreadName(c, description));
    t.setDaemon(true);
    return t;
}
```

FWIW if the thread creating the new thread is a daemon thread then new thread will be a daemon thread too.